src/ansys/dpf/core/operators/math/mode_contribution.py

```diff
@@ -0,0 +1,366 @@
+"""
+mode_contribution
+=================
+Autogenerated DPF operator classes.
+"""
+
+from warnings import warn
+from ansys.dpf.core.dpf_operator import Operator
+from ansys.dpf.core.inputs import Input, _Inputs
+from ansys.dpf.core.outputs import Output, _Outputs
+from ansys.dpf.core.operators.specification import PinSpecification, Specification
+
+
+class mode_contribution(Operator):
+    """Compute the mode contribution
+
+    Parameters
+    ----------
+    evalution_solution : FieldsContainer
+        Evalution solution
+    mode_solution : FieldsContainer
+        Mode solution
+    mode_shapes : FieldsContainer
+    result_type : str
+        Result type
+    output_dof_index : int, optional
```

Comment:
@rafacanton I'll track down who made this operator and ask for a redo of the specs